keras_nlp/models/whisper/whisper_presets.py

```diff
@@ -0,0 +1,483 @@
+# Copyright 2023 The KerasNLP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+MULTILINGUAL_SPECIAL_TOKENS = {
+    "<|startoftranscript|>": 50258,
+    "<|endoftext|>": 50257,
+    "<|notimestamps|>": 50363,
+    "<|translate|>": 50359,
+    "<|transcribe|>": 50358,
+}
+
+ENGLISH_SPECIAL_TOKENS = {
+    "<|startoftranscript|>": 50257,
+    "<|endoftext|>": 50256,
+    "<|notimestamps|>": 50362,
+    "<|translate|>": 50358,
+    "<|transcribe|>": 50357,
+}
+
+AUDIO_FEATURE_EXTRACTOR_CONFIG = {
+    "num_mels": 80,
+    "num_fft_bins": 400,
+    "stride": 160,
+    "sampling_rate": 16000,
+    "max_audio_length": 30,
+}
+
+LANGUAGE_TOKENS = {
```

Comment:
still not totally sure if we want these as assets or config, but let's keep as is for now, can revisit later.